Resources/Prototypes/_NF/Objects/Consumable/Food/meat_goblin.yml

```diff
@@ -0,0 +1,11 @@
+- type: entity
+  parent: FoodMeatRotten
+  id: FoodMeatGoblin
+  name: raw goblin meat
+  description: Just look at that marbling. Gross.
+  components:
+  - type: Sprite
+    state: rotten
+    color: lime
+  - type: Food # Frontier, New Food Quality System
+    quality: Toxin # Frontier, New Food Quality System
```

Comment:
Need to remove this to the food system PR.
Just uncomment it here and we will restore it later.